pages/wards/visits.js

```diff
@@ -55,12 +56,12 @@ export const getServerSideProps = propsWithContainer(
   verifyToken(
     async ({ authenticationToken, container }) => {
       const { wardId } = authenticationToken;
-
-      const { scheduledCalls, error } = await container.getRetrieveVisits()({
+      let { scheduledCalls, error } = await container.getRetrieveVisits()({
         wardId,
       });
+      let { ward, error: wardError } = await container.getWardById()(wardId);
       return {
-        props: { scheduledCalls, error, wardId: authenticationToken.ward },
+        props: { scheduledCalls, ward, error, wardError },
```

Comment:
As we only use error as a boolean in the render function, it's not necessary to pass both in